sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersClientIntegrationTest.java

```diff
@@ -196,6 +192,24 @@ public void beginUpdatePhoneNumberCapabilities(HttpClient httpClient) {
             .beginUpdatePhoneNumberCapabilities(phoneNumber, capabilities));
     }
 
+    private PhoneNumberCapabilities getNewPhoneCapabilities(HttpClient httpClient, String phoneNumber, String testName) {
+        PhoneNumberCapabilities newCapabilities = new PhoneNumberCapabilities();
+        PurchasedPhoneNumber acquiredNumber = this.getClientWithConnectionString(httpClient, testName).getPurchasedPhoneNumber(phoneNumber);
+
+        if (acquiredNumber.getCapabilities().getSms() == PhoneNumberCapabilityType.INBOUND_OUTBOUND) {
```

Comment:
Duplicate logic. Move this method and PhoneNumberCapabilities getNewPhoneCapabilities(PhoneNumberCapabilities capabilities) to TestBase; Call the other version getNewPhoneCapabilities(PhoneNumberCapabilities capabilities) passing in acquiredNumber.getCapabilities();

Reply:
removed duplicated logic.